packages/docs/.vitepress/config/index.ts

```diff
@@ -2,6 +2,7 @@ import { defineConfig } from 'vitepress'
 import { enConfig } from './en'
 import { sharedConfig } from './shared'
 import { zhConfig } from './zh'
+import { ukConfig } from './uk'
```

Comment:
Can you remove this? It's not needed since you are hosting the docs outside of this repo, which is easier to handle updates!

Reply:
Makes send. Closing this PR and opened a new one with a singe change only https://github.com/vuejs/pinia/pull/2422